src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -50,6 +50,16 @@
  * <pre>
  * &lt;module name="JavadocTagContinuationIndentation"/&gt;
  * </pre>
+ * <pre>
+ * &#47;**
+ *  * @tag comment..
+ *  *  Indentation spacing is 1 (violation)
+ *  *   Indentation spacing is 2 (violation)
+ *  *     Indentation spacing is 4 (OK)
```

Comment:
`(OK)` -> // OK 
Here and in other cases below

Reply:
done.